Lib/bdb.py

```diff
@@ -125,12 +187,14 @@ def dispatch_exception(self, frame, arg):
     # definition of stopping and breakpoints.
 
     def is_skipped_module(self, module_name):
+        "Return True if module_name matches any skip pattern."
```

Comment:
Thank you for explaining.  Makes sense to use single quotes for simple lines that might need to wrap.